community-operators/multicluster-operators-subscription/0.1.1/multicluster-operators-subscription.v0.1.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,629 @@
+              - command:
+                - /usr/local/bin/multicluster-operators-subscription
+                - --sync-interval=10
+                ports:
+                - containerPort: 8443                
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: multicluster-operators-hub-subscription
+                livenessProbe:
+                  exec:
+                    command:
+                    - ls
+                  initialDelaySeconds: 15
+                  periodSeconds: 15
+                readinessProbe:
+                  exec:
+                    command:
+                    - ls
+                  initialDelaySeconds: 15
+                  periodSeconds: 15
+                image: quay.io/open-cluster-management/multicluster-operators-subscription:1.0.0-SNAPSHOT-2020-03-18-20-08-33
+                imagePullPolicy: Always
+                name: multicluster-operators-hub-subscription
+                resources:
+                  limits:
+                    cpu: 500m
+                    memory: 2Gi
+                  requests:
+                    cpu: 100m
+                    memory: 128Mi
+                volumeMounts:
+                - name: tmp
+                  mountPath: "/tmp"
+                - name: multicluster-operators-subscription-tls
+                  readOnly: true
+                  mountPath: "/etc/subscription"
+              volumes:
+              - name: tmp
+                emptyDir: {}
+              - name: multicluster-operators-subscription-tls
+                secret:
+                defaultMode: 420
+                secretName: multicluster-operators-subscription
+                items:
+                - key: tls.crt
+                  path: tls.crt
+                - key: tls.key
+                  path: tls.key
+              serviceAccountName: multicluster-operators
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: true
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - Red Hat
+  - multicluster
+  - subscription
+  - channel
+  labels:
+    app: multicluster-operators-subscription
+  links:
+  - name: Multicluster Subscription Operator Project
+    url: https://github.com/open-cluster-management/multicloud-operators-subscription
+  maintainers:
+  - email: xiangli@redhat.com
+    name: Xiangjing Li
+  maturity: alpha
+  provider:
+    name: Red Hat
+  selector:
+    matchLabels:
+      app: multicluster-operators-subscription
+  replaces: multicluster-operators-subscription.v0.1.0
```

Comment:
Hi @J0zi
This is the replaces line here. Is there anything else I need to add?

Reply:
you are right:)